src/items.rs

```diff
@@ -15,7 +15,7 @@ use crate::comment::{
     FindUncommented,
 };
 use crate::config::lists::*;
-use crate::config::{BraceStyle, Config, Density, IndentStyle, Version};
+use crate::config::{BraceStyle, Config, IndentStyle, Version};
```

Comment:
The import of `Density` is missing.

Reply:
It is not needed with the changes further down in this file.

Reply:
Ah, I think you need to merge the master branch to include the change from #3586.

Reply:
Indeed thanks!